src/pipeline.ml

```diff
@@ -351,10 +348,13 @@ let tarides ?app ?notify:channel ?filter ~sched ~staging_auth () =
       docker "Dockerfile"     ["live-ci3",   "ocurrent/ci.ocamllabs.io-deployer:live-ci3",   [`Ci3 "deployer_deployer"]];
     ];
     ocurrent, "ocaml-ci", [
-      docker "Dockerfile"     ["live-engine", "ocurrent/ocaml-ci-service:live", [`Toxis "ocaml-ci_ci"]];
-      docker "Dockerfile.gitlab" ["live-engine", "ocurrent/ocaml-ci-gitlab-service:live", [`Toxis "ocaml-ci_gitlab"]];
-      docker "Dockerfile.web" ["live-www",    "ocurrent/ocaml-ci-web:live",     [`Toxis "ocaml-ci_web"];
-                               "staging-www", "ocurrent/ocaml-ci-web:staging",  [`Toxis "test-www"]];
+      docker "Dockerfile"     ["live-engine", "ocurrent/ocaml-ci-service:live", [`Ci "ocaml-ci_ci"]]
+        ~archs:[`Linux_arm64];
+      docker "Dockerfile.gitlab" ["live-engine", "ocurrent/ocaml-ci-gitlab-service:live", [`Ci "ocaml-ci_gitlab"]]
+        ~archs:[`Linux_arm64];
+      docker "Dockerfile.web" ["live-www",    "ocurrent/ocaml-ci-web:live",     [`Ci "ocaml-ci_web"];
+                               "staging-www", "ocurrent/ocaml-ci-web:staging",  [`Ci "test-www"]]
+        ~archs:[`Linux_arm64];
```

Comment:
Could we keep the Linux x86_64 builds too? It's useful for local dev work.

Reply:
I have added these for OCaml-CI.  If it is also necessary for Opam-Repo-CI it will take more work as the [Dockerfile](https://github.com/ocurrent/opam-repo-ci/blob/f18f52875ffd8192f56ab608a1faffcc8e7df16a/Dockerfile#L29) contains `[arch=XXX]`.